examples/standalone_flow_matching.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
Once you are happy with the changes, please run the notebook from beginning to end so that the execution counts neatly display 1,2,3 etc. and we can be confident that the notebook runs correctly.